code-challenges.test.js

```diff
@@ -44,8 +70,41 @@ const object3 = { number: -7 }
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+describe("randItems", () => {
+    // "randItems" is the string arguement for the describe function
+    it("takes in an array of words and returns an array with all the words capitalized", () => {
+    // the it statement says that the first letter in each element of array randomNouns1 should be capitalized
+        expect(randItems).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    // the expect statements calls the function and uses the .toEqual to compare the output
+    }) 
+}) // FAIL: ReferenceError: randItems is not defined
+describe("randItems2", () => {
+    // "randItems2" is the string arguement for the describe function
+    it("takes in an array of words and returns an array with all the words capitalized", () => {
+    // the it statement says that the first letter in each element of array randomNouns2 should be capitalized
+        expect(randItems2).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    // the expect statements calls the function and uses the .toEqual to compare the output
+    })
+}) // FAIL: ReferenceError: randItems2 is not defined
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+const randItems = randomNouns1
+    // randItems has be declare for randomNouns1
+    for (let i = 0; i < randItems.length; i++) {
+    // use a for loop to go through each element in the array
+        randItems[i] = randItems[i].charAt(0).toUpperCase() + randItems[i].slice(1)
+    // .charAt returns the first character of the string elements, then .toUpperCase capitalize the first character, then use concatenation and randItems[i].slice to combine the other characters of the word to the character capitalized
+} // PASS: randItems ✓ takes in an array of words and returns an array with all the words capitalized
+
+const randItems2 = randomNouns2
+    // randItems has be declare for randomNouns1
+    for (let i = 0; i < randItems2.length; i++) {
+    // use a for loop to go through each element in the array
+        randItems2[i] = randItems2[i].charAt(0).toUpperCase() + randItems2[i].slice(1)
+    // .charAt returns the first character of the string elements, then .toUpperCase capitalize the first character, then use concatenation and randItems2[i].slice to combine the other characters of the word to the character capitalized
+} // PASS: randItems2 ✓ takes in an array of words and returns an array with all the words capitalized
+
+// Pseudo code: Embedded. The first time doing this I was only able to capitalize all characters in the string elements. After further studying up a little, I added the .charAt() and concat with .slice() to get the pass.
```

Comment:
Great logic, though you should be creating a function. You are just creating variables and then writing code on the variable instead of creating a function.